doc/MANUAL.adoc

```diff
@@ -794,6 +794,7 @@ Examples:
 +
 * `file:///shared/nfs/directory`
 * `file:///shared/nfs/one|read-only file:///shared/nfs/two`
+* `redis://server`
```

Comment:
```suggestion
* `redis://example.com`
```

Reply:
I think that is a domain, but added it to the example